src/gason.cpp

```diff
@@ -1,8 +1,8 @@
 #include "gason.h"
 #include <stdlib.h>
 
-#define JSON_ZONE_SIZE 4096
-#define JSON_STACK_SIZE 32
+constexpr size_t JSON_ZONE_SIZE = 4096;
+constexpr size_t JSON_STACK_SIZE = 32;
```

Comment:
why not use int, as (a) that is the old type, and (b) you are casting to int anyway!